exporter/awsxrayexporter/go.mod

```diff
@@ -0,0 +1,172 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter
+
+go 1.12
+
+require (
+	cloud.google.com/go/datastore v1.0.0 // indirect
```

Comment:
Can you please make sure `go mod tidy` was run so that there are no unneeded dependencies?

Reply:
Ran tidy